src/index.ts

```diff
@@ -78,14 +79,20 @@ export async function run(): Promise<void> {
 
     core.info(`Cherry pick finished with exit code ${result.exitCode}`)
```

Comment:
let's log here the stderr and stdout for debugging

Reply:
Done